src/applications/dhp-connected-devices/components/UnauthenticatedPageContent.jsx

```diff
@@ -1,7 +1,12 @@
-import React from 'react';
```

Comment:
Sorry to butt in here, but I saw your request in platform support so I figured I'd take a peek and see if I could come up with anything. From what I can tell there are 2 issues.
1) My comment on L6
2) Your import for this component is incorrect here:
https://github.com/department-of-veterans-affairs/vets-website/blob/5d3affd8d58956e233feae4e0e8701e20ba38316/src/applications/dhp-connected-devices/containers/App.jsx#L3
It should be: 
```js
import UnauthenticatedPageContent from '../components/UnauthenticatedPageContent';
```
You need the connected version of the component so that your action creator(s) from `mapDispatchToProps` get passed into the component as props

Reply:
OMG @jerekshoe thank you so so much for the detailed explanation and for catching those issues. I was going to struggle with this for much much longer without your help. You rock!!

Reply:
I’ve definitely made both of these mistakes in the past so I know how hard they can be to catch. I’m glad that you got it working!